roles/serf/files/serf.service

```diff
@@ -1,6 +1,6 @@
 [Unit]
 Description=Serf
-After=auditd.service systemd-user-sessions.service time-sync.target
+After=network.target auditd.service systemd-user-sessions.service time-sync.target
```

Comment:
@vvb while this change is good, 

I was thinking why didn't the script fail in first place, which would have caused systemd to retry the service as well. It turns out there is a bug in the statement that install iptables rule:

```
( /sbin/iptables -L INPUT | grep {{ mdns_sport_comment }} || \
/sbin/iptables -I INPUT 1 {{ mdns_sport_rule }} )
```

should have been:

```bash
( /sbin/iptables -L INPUT | grep {{ mdns_sport_comment }} || \
  ( /sbin/iptables -I INPUT 1 {{ mdns_sport_rule }} || exit 1) )
```

we can fix this separately or in this PR, upto you.